plugins/modules/aci_syslog_remote_dest.py

```diff
@@ -0,0 +1,340 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'certified'}
+
+DOCUMENTATION = r'''
+---
+module: aci_syslog_remote_dest
+short_description: Manage Syslog Remote Destinations (syslog:RemoteDest).
+description:
+- Manage syslog remote destinations
```

Comment:
Make the description similar to short_description or longer (mainly case and class name right now)

Reply:
updated